jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.terminal
+
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.LangDataKeys
+import com.intellij.openapi.application.runInEdt
+import com.intellij.openapi.project.DumbAware
+import com.intellij.util.ExceptionUtil
+import icons.TerminalIcons
+import org.jetbrains.plugins.terminal.TerminalTabState
+import org.jetbrains.plugins.terminal.TerminalView
+import software.aws.toolkits.core.utils.error
+import software.aws.toolkits.core.utils.getLogger
+import software.aws.toolkits.jetbrains.core.credentials.AwsConnectionManager
+import software.aws.toolkits.jetbrains.core.credentials.ConnectionState
+import software.aws.toolkits.jetbrains.core.credentials.shortName
+import software.aws.toolkits.jetbrains.core.plugins.pluginIsInstalledAndEnabled
+import software.aws.toolkits.resources.message
+import software.aws.toolkits.telemetry.AwsTelemetry
+import software.aws.toolkits.telemetry.Result
+
+class OpenAwsLocalTerminal : AnAction(message("aws.terminal.action"), message("aws.terminal.action.tooltip"), TerminalIcons.OpenTerminal_13x13), DumbAware {
+
+    override fun update(e: AnActionEvent) {
+        if (!pluginIsInstalledAndEnabled("org.jetbrains.plugins.terminal")) {
```

Comment:
Not needed anymore since you gated this action in the plugin.xml

Reply:
thought @rli came across some weirdness where if it was installed but not enabled it would screw up - but again - I can test.

Reply:
conditional include through the `depends` system is guaranteed to only load if the plugin is installed and enabled.

Reply:
#win

Reply:
Updated as discussed -it's all now driven out of the plugin.xml